pkgs/development/tools/knightos/z80e/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, knightos-scas, cmake, readline, SDL2 }:
+
+stdenv.mkDerivation rec {
+  pname = "z80e";
+
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "KnightOS";
+    repo = "z80e";
+    rev = version;
+    sha256 = "18nnip6nv1pq19bxgd07fv7ci3c5yj8d9cip97a4zsfab7bmbq6k";
+  };
+
+  nativeBuildInputs = [ cmake readline SDL2 ];
```

Comment:
`readline` and `SDL2` should (most likely) be build inputs.

Reply:
Thanks. I might be confused on nativeBuildInputs ve. buildInputs. Is nativeBuildInputs not for libraries etc.?

Reply:
You should think of `nativeBuildInputs` and `buildInputs` in terms of cross-compilation. Say, you are compiling for aarch64-linux on a x86_64-linux machine. In GNU terminology aarch64-linux is the host platform and x86_64-linux the build platform. Then:

* `nativeBuildInputs` are dependencies that should be used on the *build platform*.
* `buildInputs` are dependencies that should be used on the *host platform*.

So, typically you will find things like build tools in `nativeBuildInputs` and libraries that the program uses during runtime in `buildInputs`.